app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self) -> None:
+        super().__init__("Visitor is not vaccinated")
```

Comment:
There's no need to redefine the __init__ method. Just pass the message directly when calling the exception, 
like this: `raise NotVaccinatedError("Visitor is not vaccinated")`